packages/renderless/src/tree-select/vue.ts

```
@@ -1,20 +1,28 @@
 
-export const api = ['state', 'filter', 'nodeClick', 'check']
+export const api = ['state', 'check', 'filter', 'nodeClick']
 
-export const renderless = (props, { reactive, computed, watch }, { vm, emit }) => {
+export const renderless = (props, { reactive, computed, watch, onMounted }, { vm, emit }) => {
   const api = {}
 
   const state = reactive({
-    value: computed(() => props.modelValue),
-    treeData: props.treeOp.data
+    childrenName: computed(() => (props.treeOp.props && props.treeOp.props.children) || 'children'),
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)